### PR TITLE
[DOC] Queue visibility documentation

### LINKED ISSUE #1159: Docs on using queue visibility feature for cluster queues
A new docs page for the feature: https://github.com/kubernetes-sigs/kueue/issues/168

I think it could go under the tasks page: https://kueue.sigs.k8s.io/docs/tasks/, say "Monitor pending workloads in queue status". 

This could focus on the "done" part for now - the cluster queues. It would be extended for local queues in the future.

### PR DESCRIPTION
**Pull request by stuton**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #1159

#### Does this PR introduce a user-facing change?

```release-note
added a page that describes monitoring pending workloads
```

### CODE REVIEW

File: site/content/en/docs/tasks/monitor_pending_workloads.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Monitor pending workloads"
```
Comment (mimowo) [inline site/content/en/docs/tasks/monitor_pending_workloads.md:2]: Reference the page from the main tasks page: https://kueue.sigs.k8s.io/docs/tasks/. The admin section as this is ClusterQueue.

File: site/content/en/docs/tasks/monitor_pending_workloads.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      Name:            job-sample-job-8sbgz-506c6
+      Namespace:       default
+      Name:            job-sample-job-k2bmq-44616
+      Namespace:       default
+      Name:            job-sample-job-c724j-50fd2
+      Namespace:       default
+    Last Change Time:  2023-09-28T09:22:12Z
+```
+
+The `QueueVisibility.ClusterQueues.MaxCount` parameter indicates the maximal number of pending workloads exposed in the ClusterQueue status. By default, Kueue will set this parameter to 10. When the value is set to 0, then ClusterQueues visibility updates are disabled.
```
Comment (mimowo) [inline site/content/en/docs/tasks/monitor_pending_workloads.md:81]: nit: I would break the line by sentence, so that the diffs are smaller if ever made.

**Review by mimowo — COMMENTED** [review]
/lgtm
/assign @tenzen-y 
FYI @alculquicondor

File: site/content/en/docs/tasks/monitor_pending_workloads.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+the pipeline of pending jobs, and help users to estimate when their jobs will
+start.
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+- [Kueue is installed](/docs/installation) in version 0.5.0 or later.
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/monitor_pending_workloads.md:23]:
```suggestion
- [Kueue is installed](/docs/installation) in version v0.5.0 or later.
```

File: site/content/en/docs/tasks/monitor_pending_workloads.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      Name:            job-sample-job-8sbgz-506c6
+      Namespace:       default
+      Name:            job-sample-job-k2bmq-44616
+      Namespace:       default
+      Name:            job-sample-job-c724j-50fd2
+      Namespace:       default
+    Last Change Time:  2023-09-28T09:22:12Z
+```
+
+The `QueueVisibility.ClusterQueues.MaxCount` parameter indicates the maximal number of pending workloads exposed in the ClusterQueue status.
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/monitor_pending_workloads.md:82]:
```suggestion
The `queueVisibility.clusterQueues.maxCount` parameter indicates the maximal number of pending workloads exposed in the ClusterQueue status.
```

Also, I think that we should mention where users can set these parameters (`Configuration` resource).

File: site/content/en/docs/tasks/monitor_pending_workloads.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+By default, Kueue will set this parameter to 10. 
+When the value is set to 0, then ClusterQueues visibility updates are disabled.
+
+```yaml
+    queueVisibility:
+      clusterQueues: 
+        maxCount: 0
+```
+
+The `QueueVisibility.UpdateIntervalSeconds` parameter allows to control the period of snapshot updates after Kueue startup.
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/monitor_pending_workloads.md:92]:
```suggestion
The `queueVisibility.updateIntervalSeconds` parameter allows to control the period of snapshot updates after Kueue startup.
```

File: site/content/en/docs/tasks/monitor_pending_workloads.md (hunk 4 above)
Comment (stuton) [reply site/content/en/docs/tasks/monitor_pending_workloads.md]: done https://github.com/kubernetes-sigs/kueue/pull/1168/commits/077f5d7ee0e4fe7492dd85db4013f813115c1d76

**Review by mimowo — COMMENTED** [review]
/lgtm

**Review by tenzen-y — COMMENTED** [review]
/approve

**Comment by tenzen-y** [comment]
@stuton Ah, this PR's target branch is website-release-0.5. Can you change the target branch to the main?

**Comment by tenzen-y** [comment]
Recently, we changed the branch operation policy for the website.

**Comment by alculquicondor** [comment]
Please target the `main` branch.

**Comment by stuton** [comment]
changed to `main` branch

**Pull request merged**